src/test/java/core/basesyntax/service/PrecessDataServiceTest.java

```diff
@@ -0,0 +1,63 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.OperationProcessorImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.handler.BalanceHandler;
+import core.basesyntax.strategy.handler.OperationHandler;
+import core.basesyntax.strategy.handler.PurchaseHandler;
+import core.basesyntax.strategy.handler.ReturnHandler;
+import core.basesyntax.strategy.handler.SupplyHandler;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class PrecessDataServiceTest {
+    private static OperationStrategy operationStrategy;
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
```

Comment:
you don't use these variables in the tests, only in the BeforeAll method, maybe it's better to move them there